packages/embeds/js/src/features/blocks/bubbles/video/components/VideoBubble.tsx

```diff
@@ -77,9 +77,12 @@ export const VideoBubble = (props: Props) => {
               }
             >
               <video
-                autoplay={props.onTransitionEnd ? false : true}
+                autoplay={
+                  (props.onTransitionEnd ? false : true) ||
+                  props.content?.hideControls
+                }
```

Comment:
I decided to autoplay the video as it would be impossible to play the video without controls.

Reply:
This check is here to avoid replaying all the media bubbles when we return to a chat session (after a refresh or else).

So this is need to stay as is otherwise, all the media without controls would play all at once on a page refresh if the chat state is saved